### PR TITLE
Nhj input flex

### PR DESCRIPTION
**Pull request by NHJohnson**
## Description

This is adding some more flexibility to the inputs of the somatic consensus workflow. 

Fixes # https://github.com/d3b-center/bixu-tracker/issues/1021

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)
- [X] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Environment:
* Test files:

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings
- [ ] I have committed any related changes to the PR

### CODE REVIEW

File: workflow/kfdrc_consensus_calling.cwl [hunk 1]
```diff
@@ -19,6 +19,11 @@ inputs:
   output_basename: string
   annotation_vcf: {type: File, secondaryFiles: ['.tbi'], doc: "VCF of annotions to add to consensus variants, e.g. gnomAD allele frequency"}
   vep_cache: {type: File, doc: "tar gzipped cache from ensembl/local converted cache"}
+  annot_columns: {type: string?, default: 'INFO/AF', doc: "column from annotation_vcf to add to consensus VCF"}
+  filter_names: {type: 'string[]?', default: [ "NORM_DP_LOW", "GNOMAD_AF_HIGH" ], doc: "Names of filters to be added to consensus VCF"}
+  depth_lowerbound: {type: string?, default: '7', doc: "Normal-sample read depth at which to apply depth filter"}
+  frequency_upperbound: {type: string?, default: '0.001', doc: "Population allele frequency above which to apply frequency filter"}
```
Comment (migbro) [inline workflow/kfdrc_consensus_calling.cwl:25]: This should be changed to float, and value should be `0.001` with no quotes. In the context of passing to the tool itself in the workflow, it'll still be interpreted correctly.

File: workflow/kfdrc_consensus_calling.cwl [hunk 2]
```diff
@@ -92,8 +96,7 @@ steps:
       tumor_id: input_tumor_name
       output_basename: output_basename
       reference: indexed_reference_fasta
-      retain_info:
-        valueFrom: ${return "MQ,MQ0,CAL,HotSpotAllele"}
+      retain_info: maf_retain_info
```
Comment (migbro) [inline workflow/kfdrc_consensus_calling.cwl:99]: I am also working on adding an optional `retain_fmt` input, but that should probably be a seperate PR

**Review by migbro — CHANGES_REQUESTED** [review]
We can talk about this a bit this afternoon, your approach addresses most of what I was looking for, but also ends up taking away a bit of flexibility in the `variant_filter` step for the `filter_expression` inputs

**Comment by NHJohnson** [comment]
Most of the desired features of this PR got integrated into a different PR and merged, so I'm going to delete this one.

**Pull request closed without merging**